Gemfile.lock

```diff
@@ -17,6 +17,7 @@ GEM
     ethon (0.9.1)
       ffi (>= 1.3.0)
     ffi (1.9.14)
+    ffi (1.9.14-x64-mingw32)
```

Comment:
I didn't notice this before but I'm guessing these -x64-mingw32 gem versions showed up due to bundle installing on Windows?  We likely do not need to commit these Gemfile.lock changes